From 041f0d92632417d8ce7361ec2a874cfacd29ed07 Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Thu, 7 Apr 2016 20:28:33 +0300
Subject: [PATCH 1020/1240] net: mvpp2x: add eth2addr environment variable to
 default config

- Added eth2addr environment variable to default configuration
- Armada-70x0 has 3 network interfaces and needs to configure 3 MAC-Address

Change-Id: I64c2fef1dad94b366b0674af3338b459a1f09064
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28892
Reviewed-by: Haim Boot <hayim@marvell.com>
Tested-by: Haim Boot <hayim@marvell.com>
---
 include/configs/mvebu-common.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/mvebu-common.h b/include/configs/mvebu-common.h
index e06c6f6..c3da21b 100644
--- a/include/configs/mvebu-common.h
+++ b/include/configs/mvebu-common.h
@@ -382,6 +382,8 @@
 	#define CONFIG_ETHADDR          00:00:00:00:51:81
 	#define CONFIG_HAS_ETH1
 	#define CONFIG_ETH1ADDR		00:00:00:00:51:82
+	#define CONFIG_HAS_ETH2
+	#define CONFIG_ETH2ADDR		00:00:00:00:51:83
 	#define ENV_ETH_PRIME           "e1000#0"
 	#define CONFIG_PHY_GIGE			/* Include GbE speed/duplex detection */
 
-- 
1.9.1

